Gruntfile.js

```diff
@@ -30,6 +30,7 @@ module.exports = function(grunt) {
     shell: {
       test: {
         command : '<%= config.binPath %>/istanbul cover --report lcov --dir test/reports/ <%= config.binPath %>/_mocha test/spec -- --reporter xunit-file',
+        //command : '<%= config.binPath %>/istanbul cover --report lcov --dir test/reports/ <%= config.binPath %>/_mocha test/spec -- --reporter spec',
```

Comment:
This could be rewritten to support a grunt.option to switch the reporting instead of having to comment/uncomment code a necessary, i'll be happy to show you how to do that